admin_guide/cluster_autoscaler.adoc

```diff
@@ -0,0 +1,527 @@
+[[admin-guide-cluster-autoscaler]]
+= Managing the Cluster Auto Scaler
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[managing-autoscaler-overview]]
+== Overview
+
+This topic describes setting up the auto-scaler on your
+{product-title} cluster. The auto-scaler's remit is to ensure your
+pods have somewhere to run, and to also ensure there are no unneeded
+nodes when demand drops.
+
+Note: this feature is currently in tech-preview and the this guide
```

Comment:
Will an autoscaler be available for other clouds, or is this an AWS-only feature?

Reply:
Yes, the plan is: AWS, Azure, OpenStack, then input from PM.

Reply:
We'll need to talk about the cadence for releases to other clouds.